app/controllers/users_controller.rb

```diff
@@ -1,13 +1,10 @@
 class UsersController < ApplicationController
   def index
-    if current_user
-      @users = User.all
-    else
-      redirect_to user_session_path
-    end
+    @users = User.all
```

Comment:
- [ ] please verify the user can log in. now can't

![kkkk](https://user-images.githubusercontent.com/61562626/148540807-454e1294-11d8-4d86-b449-c07db59da284.jpeg)

Reply:
Fixed